README.md

```diff
@@ -487,7 +487,7 @@ interface Foo {
 }
 ```
 
-### Key Definition
+### Dynamic Property Keys
```

Comment:
I wouldn't mind. But if you keep "dynamic" you will have to fix #97 so that the title somehow matches.